laterpay/asset_sources/scss/blocks/premium-download-box.scss

```diff
@@ -0,0 +1,123 @@
+// shortcodes for linked premium files in free to read posts -----------------------------------------------------------
+.wp-block-laterpay-premium-download-box.lp_premium-file-box {
+	background: $background--darker url('../img/premium-text.png') 50% 50% no-repeat;
+	background-size: cover !important;
```

Comment:
`!important` is always discouraged, add one liner comment as to why you had to use it, if you must